types/watch/index.d.ts

```diff
@@ -1,35 +1,37 @@
-// Type definitions for watch
+// Type definitions for watch 1.0
 // Project: https://github.com/mikeal/watch
-// Definitions by: Carlos Ballesteros Velasco <https://github.com/soywiz>
+// Definitions by: Carlos Ballesteros Velasco <https://github.com/soywiz>, Gyusun Yeom <https://github.com/Perlmint>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 // Imported from: https://github.com/soywiz/typescript-node-definitions/watch.d.ts
 
 /// <reference types="node" />
 
-
 import fs = require("fs");
 import events = require("events");
 
+export type FileOrFiles = fs.Stats | {[key: string]: fs.Stats};
+
 export interface Monitor extends events.EventEmitter {
-    // event: created
-    // event: removed
-    // event: changed
+    files: {[key: string]: fs.Stats};
 
-    // export function onCreated(callback, function(f, stat: fs.Stats) { });
-    // export function onChanged(callback, function(f, curr: fs.Stats, prev: fs.Stats) { });
-    // export function onRemoved(callback, function(f, stat: fs.Stats) { });
+    on(event: "created" | "removed", listener: (f: FileOrFiles, stat: fs.Stats) => void): this;
```

Comment:
@Perlmint This is wrong. https://github.com/mikeal/watch/blob/master/main.js#L139 Emits a `filename: string` as first parameter.